docs/guide/actions.md

```diff
@@ -16,9 +16,9 @@ Let's write an app with a simple action.
 --8<-- "docs/examples/guide/actions/actions01.py"
 ```
 
-The `action_set_background` method is an action which sets the background of the screen. The key handler above will call this action if you press the ++r++ key.
+The `action_set_background` method is an action method that sets the background of the screen. The key handler above will call this action if you press the ++r++ key.
```

Comment:
A method isn't an action, and sloppy usage will inevitably lead to misunderstanding on the part of at least some readers.